docs/user_guide/flows.rst

```diff
@@ -1,66 +1,73 @@
 Flows
-===================================
+=====
 
-SiliconCompiler flows are created by configuring the 'flowgraph' parameters within the schema. To simplify reuse of complex flows, the project includes standardized interfaces for bundling flowgraph settings as reusable named modules. These flow modules get loaded by the :meth:`.load_flow()` function during compilation setup. A complete set of supported open PDKs can be found in the :ref:`Flows Directory`. The table below shows the function interfaces required in setting up flows.
-
-
-.. list-table::
-   :widths: 10 10 10 10 10 10
-   :header-rows: 1
-
-   * - Function
-     - Description
-     - Arg
-     - Returns
-     - Used by
-     - Required
-
-   * - **setup**
-     - PDK setup function
-     - chip
-     - chip
-     - load_flow()
-     - yes
-
-   * - **make_docs**
-     - Doc generator
-     - None
-     - chip
-     - sphinx
-     - yes
+SiliconCompiler flows are created by configuring the 'flowgraph' parameters within the schema. To simplify reuse of complex flows, the project includes standardized interfaces for bundling flowgraph settings as reusable named modules.
 
+Similar to other types of SiliconCompiler modules, flows are loaded by passing a :class:`.Flow` object into the :meth:`.use()` function before a run is started.
 
 setup(chip)
 -----------------
 
 A SiliconCompiler flowgraph consists of a set of connected nodes and edges, where a node is an executable tool performing some ("task"), and an edge is the connection between those tasks. The first task in the flowgraph must be named 'import'. ::
 
-  chip.node(flow, 'import', <import_tool>)
-  chip.node(flow, <step>, <step_tool>)
-  chip.edge(flow, 'import', <step>)
+  flow.node(flow, 'import', <import_tool>)
+  flow.node(flow, <step>, <step_tool>)
+  flow.edge(flow, 'import', <step>)
```

Comment:
the signature of .node is <flow>, <step>, <tool>, <task>, so these should be updated to reflect that